### PR TITLE
Fix golint issues in client/custom_metrics/scheme

### PR DESCRIPTION
**Pull request by lanzao**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

Ref #68026
**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:
I have run hack/update-bazel.sh and hack/update-gofmt.sh to make sure I fix any problems with gofmt or bazel.
@dims 

**Release note:**

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/metrics/pkg/client/custom_metrics/scheme/register.go [hunk 1]
```diff
@@ -26,12 +26,19 @@ import (
 	cmv1beta2 "k8s.io/metrics/pkg/apis/custom_metrics/v1beta2"
 )
 
+// GroupName is the group name use in this package.
 const GroupName = cmv1beta1.GroupName
 
+// SchemeGroupVersion is group version used to register these objects.
 var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: runtime.APIVersionInternal}
 
+// Scheme is the runtime.Scheme to which all cmint api types are registered.
```
Comment (RainbowMango) [inline staging/src/k8s.io/metrics/pkg/client/custom_metrics/scheme/register.go:35]: What's the `cmint` mean? Is there a typo?

Comment (lanzao) [reply staging/src/k8s.io/metrics/pkg/client/custom_metrics/scheme/register.go]: This is a poor abbreviation of “custom metrics api” and has been corrected.

**Comment by RainbowMango** [comment]
/lgtm
/approve

@lanzao Congrats! Seems this is your first PR.

There is something you need to know that we are going to [remove golint based verification](https://github.com/kubernetes/kubernetes/pull/98063).

**Pull request merged**